gnucash/register/ledger-core/split-register.c

```diff
@@ -1223,6 +1223,7 @@ gnc_split_register_delete_current_trans (SplitRegister* reg)
         xaccTransCommitEdit (trans);
     }
     gnc_resume_gui_refresh();
+    gnc_split_register_redraw (reg);
```

Comment:
@Bob-IT Does it make sense to force a redraw or would it be better to move the `gnc_resume_gui_refresh() to before `if (was_open)`?

Reply:
just FYI I tried this
```
    was_open = xaccTransIsOpen (trans);
    xaccTransDestroy (trans);
    gnc_resume_gui_refresh();
    if (was_open)
    {
        DEBUG ("committing");
        xaccTransCommitEdit (trans);
    }
```
But this does not fix the issue, the split is still un-editable.

Reply:
OK, and looking at the rest of the file,
```
gnc_resume_gui_refresh();
gnc_split_register_redraw (reg);
```
is used in 3 other places. We'll go with it.

Reply:
That was also my reasoning, but I have to admit, I don't really know why/how it fixes the issue. I do know that the GUID for the blank split is invalid after the delete, and that's what cause the split to become un-editable, but I don't really understand which part of gnc_split_register_redraw fixes that (but it does)...

Reply:
The register holds a dangling guid reference to the old blank split. Redrawing the register causes it to destroy those cells and make a new empty transaction with a new blank split